packages/shared/src/components/buttons/common.ts

```diff
@@ -210,7 +210,6 @@ export const useGetIconWithSize = (
       className: classNames(
         icon.props.className,
         !iconOnly && 'text-base',
-        !iconOnly && !icon.props?.size && '!h-6 !w-6',
```

Comment:
Are you sure we didn't add this recently for some other thing 😅
I could have sworn someone else just touched this.

Reply:
Yeah, we should not need it. There is always a fallback size for our icons. Otherwise, we won't need the `!` important prefixes.